sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md

```diff
@@ -10,6 +10,9 @@
 
 ### Other Changes
 
+* Remove metric namespace mapping.
```

Comment:
nit: Would be nice if we describe what changes the user notices with this change.